### PR TITLE
build(docs-infra): redirect removed webpack guide to `v5.angular.io`

### PR DESCRIPTION
**Pull request by gkalpak**
The outdated webpack guide has been removed in #24478, but people might still try to access it (via direct links or search-engine results).

Instead of returning 404, we will now redirect `/guide/webpack` to the archived version of the guide at `v5.angular.io/guide/webpack`.

### CODE REVIEW

File: aio/tests/deployment/shared/URLS_TO_REDIRECT.txt [hunk 1]
```diff
@@ -178,7 +178,8 @@
 /guide/service-worker-getstart	/guide/service-worker-getting-started
 /guide/service-worker-comm	/guide/service-worker-communications
 /guide/service-worker-configref	/guide/service-worker-config
+/guide/webpack	https://v5.angular.io/guide/webpack
 /news	https://blog.angular.io/
 /news.html	https://blog.angular.io/
 /testing	/guide/testing
-/testing/first-app-tests.html	/guide/testing
\ No newline at end of file
+/testing/first-app-tests.html	/guide/testing
```
Comment (petebacondarwin) [inline aio/tests/deployment/shared/URLS_TO_REDIRECT.txt]: what changed on this line?

**Review by petebacondarwin — APPROVED** [review]
LGTM

**Comment by petebacondarwin** [comment]
Actually the AIO job on Travis looks like a valid failure related to this change...

File: aio/tests/deployment/shared/URLS_TO_REDIRECT.txt (hunk 1 above)
Comment (gkalpak) [reply aio/tests/deployment/shared/URLS_TO_REDIRECT.txt]: Oh, it's my editor ensuring there's a newline at the end (as it should), which causes the tests to break.
I'll fix it 😇

Comment (petebacondarwin) [reply aio/tests/deployment/shared/URLS_TO_REDIRECT.txt]: But normally in that case GitHub shows a little icon to show there is no trailing newline. But not in this case...

Comment (gkalpak) [reply aio/tests/deployment/shared/URLS_TO_REDIRECT.txt]: True. Maybe not for `txt` files?

Comment (gkalpak) [reply aio/tests/deployment/shared/URLS_TO_REDIRECT.txt]: Fixed. Waiting for Travis to be green.

**Pull request closed without merging**